apps/mudita-center-e2e/src/specs/stress-tests/connected-devices-stress-test.ts

```diff
@@ -0,0 +1,337 @@
+import { E2EMockClient } from "../../../../../libs/e2e-mock/client/src"
+import {
+  overviewDataWithOneSimCard,
+  overviewDataWithOneSimCard2nd,
+  overviewDataWithOneSimCard3rd,
+  overviewDataWithOneSimCard4th,
+  overviewDataWithOneSimCard5th,
+  overviewDataWithOneSimCard6th,
+} from "../../../../../libs/e2e-mock/responses/src"
+import OverviewKompaktPage from "../../page-objects/overview-kompakt.page"
+import HomePage from "../../page-objects/home.page"
+import { kompaktImageRegex } from "../../consts/regex-const"
+import overviewPage from "../../page-objects/overview.page"
+import drawerPage from "../../page-objects/drawer.page"
+
+describe("Kompakt switching devices", () => {
+  const firstSerialNumber = "KOM1234567890"
+  const secondSerialNumber = "KOM1234567892"
+  const thirdSerialNumber = "KOM1234567893"
+  const fourthSerialNumber = "KOM1234567894"
+  const fifthSerialNumber = "KOM1234567895"
+  const sixthSerialNumber = "KOM1234567896"
+
+  before(async () => {
+    E2EMockClient.connect()
+    await browser.waitUntil(() => {
+      return E2EMockClient.checkConnection()
+    })
+  })
+
+  after(() => {
+    E2EMockClient.stopServer()
+    E2EMockClient.disconnect()
+  })
+
+  it("Connect 1,2 Kompakt devices", async () => {
+    const devices = [
+      {
+        path: "path-1",
+        body: overviewDataWithOneSimCard,
+        serialNumber: firstSerialNumber,
+      },
+      {
+        path: "path-2",
+        body: overviewDataWithOneSimCard2nd,
+        serialNumber: secondSerialNumber,
+      },
+    ]
+
+    for (const device of devices) {
+      E2EMockClient.mockResponse({
+        path: device.path,
+        body: device.body,
+        endpoint: "FEATURE_DATA",
+        method: "GET",
+        status: 200,
+      })
+      E2EMockClient.addDevice({
+        path: device.path,
+        serialNumber: device.serialNumber,
+      })
+
+      await browser.pause(6000)
+    }
+  })
+
+  it("Check Drawer Modal and Switch to 1st device", async () => {
+    const deviceSelectDrawer = await drawerPage.deviceSelectDrawer
+    await expect(deviceSelectDrawer).toBeDisplayed()
```

Comment:
I would call this test and change variables to something more fitting - this is Connected Devices Page / Modal - this is not a Drawer Page. How would next test where there will be an actual drawer ?


`Mudita Center.2.3.1-mockdev.45.app`

🔴  Test fails on first to phones 
```
Error: Expect $(`[data-testid="device-select-drawer-content"]`) to be displayed
Error: Expect $(`[data-testid="generic-view-image"]`) to be displayed
Error: Expect $(`//*[@data-testid="drawer-device-wrapper-KOM1234567892"]`) to be displayed
```

```
Expected: "displayed"
Received: "not displayed"
    at /Users/tomasz/Dev/Mudita-Center/mudita-center/apps/mudita-center-e2e/src/specs/stress-tests/connected-devices-stress-test.ts:69:38
    at step (/Users/tomasz/Dev/Mudita-Center/mudita-center/apps/mudita-center-e2e/src/specs/stress-tests/connected-devices-stress-test.ts:33:23)
    at Object.next (/Users/tomasz/Dev/Mudita-Center/mudita-center/apps/mudita-center-e2e/src/specs/stress-tests/connected-devices-stress-test.ts:14:53)
    at fulfilled (/Users/tomasz/Dev/Mudita-Center/mudita-center/apps/mudita-center-e2e/src/specs/stress-tests/connected-devices-stress-test.ts:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

Reply:
![Screenshot 2024-08-12 at 8 50 39 AM](https://github.com/user-attachments/assets/3d62463e-3dff-4f52-8d59-9cc13220acb5)